src/UI Component/Login/login.js

```diff
@@ -0,0 +1,121 @@
+
+import React, { useState } from "react";
+import SignUpPage from "./signuppage"
+import AdminDashboard from "../Adminpanel/dashboard";
+import UserDashBoard from "../User/userdashboard";
+
+
+const SignIn = () => {
+  const [formInput, setFormInput] = useState({
+    email: "",
+    password: "",
+  });
+  const [isSignUp, setIsSignUp] = useState(false);
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormInput({ ...formInput, [name]: value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    // Check if the entered credentials are valid
+    const adminCredentials = {
+      email: "admin@mail.com",
+      password: "password123",
+    };
+
+    const userCredentials = {
+      email: "ephy@mail.com",
+      password: "password123",
+    };
+
+    if (
+      formInput.email === adminCredentials.email &&
+      formInput.password === adminCredentials.password
+    ) {
+      setIsLoggedIn(true);
+    } else if (
+      formInput.email === userCredentials.email &&
+      formInput.password === userCredentials.password
+    ) {
+      setIsLoggedIn(true);
+    } else {
+      alert("Invalid credentials. Please try again.");
+    }
+  };
+
+  const handleSignUpClick = () => {
+    setIsSignUp(true);
+  };
+
+  const handleBackToLoginClick = () => {
+    setIsSignUp(false);
+  };
+
+  return (
+    <div className="">
+      {isLoggedIn ? (
+        formInput.email === "admin@mail.com" ? (
```

Comment:
Please don't check the condition here. 

Create another state to hold the result of a check for if the user is admin or not. Then use that state value here.